integration_tests/run_tests.py

```diff
@@ -82,6 +82,8 @@ def get_args():
                 help="Run supported tests with --fast")
     parser.add_argument("-m", action='store_true',
                 help="Check that all module names are unique")
+    parser.add_argument("-experimental-simplifier", "---experimental_simplifier",
```

Comment:
Not sure what is going on here, but for longer options we should just use two dashes `--`:
```suggestion
    parser.add_argument("--experimental-simplifier",
```

Reply:
yeah that's right. It was a typo from me.
so the convention is `-` for single character `--` for words or long flag names.

Reply:
Resolved